Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: test
+
+test: database-v1
+	@go test -v -race ./...
+
+database-v1:
```

Comment:
that would work. suggestions for folders? It'll be a mess to have them at the root of the repository.